src/KubeLibrary/KubeLibrary.py

```diff
@@ -611,6 +611,27 @@ def get_secrets_in_namespace(self, name_pattern, namespace, label_selector=""):
         secrets = [item for item in ret.items if r.match(item.metadata.name)]
         return secrets
 
+    def get_namespaced_pod_exec(self, name, namespace, cmd):
+        """Exec command on selected container for POD.
+
+        Returns command stdout/stderr
+
+        - ``name``:
+          pod name
+        - ``namespace``:
+          namespace to check
+        - ``command``:
+          command to be executed
+        """
+        return stream.stream(self.v1.connect_get_namespaced_pod_exec,
+                             name,
+                             namespace,
+                             command=["/bin/sh", "-c", cmd],
+                             stderr=True,
```

Comment:
how to get stderr vs stdout?

Reply:
it always returns a string value, this flags parameters enable it or disable if we do not need stderr, stdout, etc